HierarchicalDataConverter/Catalyst.HierarchicalDataConverter/Catalyst.HierarchicalDataConverter.csproj

```diff
@@ -40,6 +40,9 @@
     <Reference Include="Catalyst.DataProcessing.Shared.Utilities, Version=4.4.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\Catalyst.DataProcessing.Shared.Utilities.4.4.1812.1201\lib\net461\Catalyst.DataProcessing.Shared.Utilities.dll</HintPath>
     </Reference>
+    <Reference Include="Catalyst.Platform.CommonExtensions, Version=1.0.0.0, Culture=neutral, PublicKeyToken=e99707dc63721e10, processorArchitecture=MSIL">
```

Comment:
We are trying to move away from using Catalyst.Platform.CommonExtensions. This functionality has been migrated to Catalyst.DataProcessing.Shared.Utilities.

Reply:
You need to use the Catalyst.Platform.CommonExtensions for it to work in 18.2